src/setupJest.ts

```diff
@@ -1,4 +1,5 @@
 import 'jest-preset-angular';
+import 'jest-extended';
```

Comment:
Why do you need `jest-extended` exactly?

Reply:
when before i started messing with the typings, i thought i needed this for some missing functions off of jest... now testing locally, everything seems to work without it, so will remove